tokio/src/lib.rs

```diff
@@ -471,6 +471,7 @@ compile_error!("The `tokio_taskdump` feature requires `--cfg tokio_unstable`.");
 
 #[cfg(all(
     tokio_taskdump,
+    not(doc),
```

Comment:
small drive by—I wanted to make it so docs would build on any platform

Reply:
Thanks, but can you file that as a separate PR?